src/NuProj.ProjectSystem.12/Templates/Projects/NuProj/NuProj.nuproj

```diff
@@ -32,4 +32,10 @@
     <Content Include="Readme.txt" />
   </ItemGroup>
   <Import Project="$(NuProjPath)\NuProj.targets" />
+  <!-- To modify your build process, add your task inside one of the targets below and uncomment it. 
+  <Target Name="BeforeBuild">
+  </Target>  
+  <Target Name="AfterBuild">
+  </Target>
+  -->
```

Comment:
Personally I'm not a fan of this extra xml. It trails most projects and most projects don't do anything with them. Can we leave this off the template?